### ISSUE TITLE
Publish "Collector Bootcamp" as a public repository

### ISSUE THREAD

**Issue by BunnyNabbit** [BunnyNabbit/node-celaria-server#21]
A good library should have good examples. Collector Bootcamp is a server made using node-celaria-server. Actually, it isn't even versioned, it just exists zhere on Soft Silver, which hasn't been moved to Feral Hare yet. Some issues I'll have to resolve internally:
- Move Discord tokens to configuration files or variables.
- Document database usage.
- Use zhe `mongodb` package instead of `mongojs`.
For zhe Discord integration, it does use a profile generator. I might need to include `express` and run a http service on top of it.

**Comment by BunnyNabbit** [comment]
published under bunnynabbit/collector-bootcamp
